css/components/button.css

```diff
@@ -0,0 +1,146 @@
+.btn {
+  background: transparent;
+  border: none;
+  outline: none;
+  display: inline-block;
+  border: 1px var(--light) solid;
+  border-radius: var(--rounded);
+  font-size: 1rem;
+  padding: 0.4em 0.8em;
+  margin: 0.1em;
+  cursor: pointer;
+}
+
+.btn-sm {
+  font-size: 0.8rem;
+}
+
+.btn-lg {
+  font-size: 1.2rem;
+}
+
+.disabled {
+  opacity: 0.5;
+  pointer-events: none;
+}
+
+.btn-primary,
+.btn-danger,
+.btn-info,
+.btn-secondary,
+.btn-success,
+.btn-warning,
+.btn-light,
+.btn-dark,
+.btn-white {
+  border: none;
+}
+
+.btn-outline-primary,
+.btn-outline-danger,
+.btn-outline-info,
+.btn-outline-secondary,
+.btn-outline-success,
+.btn-outline-warning,
+.btn-outline-light,
+.btn-outline-dark {
+  background-color: var(--white);
+  transition: background-color 0.3s linear;
+}
```

Comment:
have a utility class for this

Reply:
I am applying light border to button in **btn** class itself so that even without any other classes like **btn-primary** or **btn-success**  applied it will look like a button and with these classes applied I don't want that light border around it that's why I added border:none to all classes there
And we will always have to write utility class name **border-none** with these classes if utility class used instead of this
I maybe wrong but can it be done in any other way

Reply:
If you are following this approach it's okay, I was just suggesting that with utility classes things get more flexible